### DISCUSSION TITLE
Your Contributions

### DISCUSSION THREAD

**Discussion by mohamad-wd10** [community/community#48915] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

GitHub commits aren't recorded in the 'Your Contributions` calendar

**Comment by radar-blip** [comment]
You might find some useful info here: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

**Comment by znetworkx** [comment]
OK
